lunavl/sdk/detectors/facedetector.py

```diff
@@ -114,6 +114,11 @@ def asDict(self) -> Dict[str, Any]:
             )
         return res
 
+    @property
+    def detection(self) -> DetectionFloat:
+        """Get core detection"""
+        return self._coreEstimation.detection
```

Comment:
for what?

Reply:
Because using private variables in user code is wrong. `medicalMaskEstimator.estimate(image, faceDetection.detection)` is better then `medicalMaskEstimator.estimate(image, faceDetection._coreEstimation.detection`

Reply:
thre is coreEstimation

Reply:
I don't understand. I added this method because it was using a private variable. I think it's better than a direct call

Reply:
there is property `coreEstimation`

Reply:
Done